### PR TITLE
SQL Items mods added Skirmish +1

### PR DESCRIPTION
**Pull request by some1here**

### CODE REVIEW

**Comment by maxtherabbit** [comment]
The Travis CI build could not complete due to an error

**Comment by maxtherabbit** [comment]
oops travis is broke

File: sql/item_mods.sql [hunk 1]
```diff
@@ -25220,4 +25220,520 @@ INSERT INTO `item_mods` VALUES (28665, 161, -8); -- Physical damage taken -8%
...
+INSERT INTO `item_mods` VALUES (28298, 9, 13);
+INSERT INTO `item_mods` VALUES (28298, 10, 9);
+INSERT INTO `item_mods` VALUES (28298, 11, 32);
+INSERT INTO `item_mods` VALUES (28298, 13, 10);
+INSERT INTO `item_mods` VALUES (28298, 14, 27);
+INSERT INTO `item_mods` VALUES (28298, 23, 5);
+INSERT INTO `item_mods` VALUES (28298, 68, 69);
+INSERT INTO `item_mods` VALUES (28298, 31, 64);
+INSERT INTO `item_mods` VALUES (28298, 29, 3);
+INSERT INTO `item_mods` VALUES (28298, 384,'51);
```
Comment (xdemolish) [inline sql/item_mods.sql]: hay thar rouge '

**Comment by R3P0FFXI** [comment]
A lot of these mods are incorrect, first item I looked at haste mod is incorrect

INSERT INTO `item_mods` VALUES (27878, 384, 300);

Also you have 0 comments so no one will have any idea what half these mods are

**Comment by TeoTwawki** [comment]
Haste is in fraction of 1024, not true percentage. 
Example: 51/1024 = 5% so MOD_HASTE_GEAR of 51
它是在1024份 而不是真正的百分比。
例如：一千〇二十四分之五十一= 5％
如此 51 MOD_HASTE_GEAR

**Comment by some1here** [comment]
correction was added

**Pull request closed without merging**